src/components/Mypage/EditNickName.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useEffect } from 'react';
+import * as S from './styles';
+
+const EditNickName = ({ userNickName }) => {
+  const [nickName, setNickName] = useState('');
+  const EditName = () => {

```

Comment:
이건 왜 또 함수명이 PascalCase죠?

```suggestion
  const editName = () => {
```

Reply:
4ebf5cf